pkg/phlare/modules.go

```diff
@@ -364,12 +365,15 @@ func (f *Phlare) context() context.Context {
 func (f *Phlare) initIngester() (_ services.Service, err error) {
 	f.Cfg.Ingester.LifecyclerConfig.ListenPort = f.Cfg.Server.HTTPListenPort
 
-	ingester, err := ingester.New(f.context(), f.Cfg.Ingester, f.Cfg.PhlareDB, f.storageBucket, f.Overrides)
+	svc, err := ingester.New(f.context(), f.Cfg.Ingester, f.Cfg.PhlareDB, f.storageBucket, f.Overrides)
 	if err != nil {
 		return nil, err
 	}
-	ingesterv1connect.RegisterIngesterServiceHandler(f.Server.HTTP, ingester, f.auth)
-	return ingester, nil
+	ingesterv1connect.RegisterIngesterServiceHandler(f.Server.HTTP, svc, f.auth)
+	//pyroscopePath := "/pyroscope/ingest"
```

Comment:
@cyriltovena @simonswine Not sure how you all feel about the prefix? I know /render and /label-values [have the prefix](https://github.com/grafana/phlare/pull/591/files#diff-ff08860a084147a57eed45febf9519697bc6179c98981010040b66ffde458e79R208-R209)

I don't feel strongly one way or the other.

Reply:
Fine for me

Reply:
FWIW, I added `/pyroscope` prefix [here](https://github.com/grafana/backend-enterprise/pull/4418/files#diff-ef48fc318181e7feb0e84c93de69f745eec72eb9450764e97a15866530656c88R370) for consistency, so I guess now I do lean slightly more towards having the prefix.